[2020-01-07 22:37:29,245] [app.py:36] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:45:18,725] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:45:51,277] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:47:05,555] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:05,556] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:05,587] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:05,588] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:05,604] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:05,604] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:05,950] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:05,951] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:05,970] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:05,971] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:05,988] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:05,989] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:11,019] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:11,019] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:11,044] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:11,046] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:11,065] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:11,065] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:41,133] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:41,135] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:41,157] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:41,158] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:47:41,180] [views.py:31] [INFO]	got you at index
[2020-01-07 22:47:41,180] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:48:41,219] [views.py:31] [INFO]	got you at index
[2020-01-07 22:48:41,220] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:48:41,254] [views.py:31] [INFO]	got you at index
[2020-01-07 22:48:41,254] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:48:41,283] [views.py:31] [INFO]	got you at index
[2020-01-07 22:48:41,285] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:51:09,124] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:51:17,047] [views.py:31] [INFO]	got you at index
[2020-01-07 22:51:17,047] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:51:47,368] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:51:50,535] [views.py:31] [INFO]	got you at index
[2020-01-07 22:51:50,536] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:53:34,453] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:53:36,306] [views.py:31] [INFO]	got you at index
[2020-01-07 22:53:36,310] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:55:35,447] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 22:55:38,339] [views.py:31] [INFO]	got you at index
[2020-01-07 22:55:38,339] [views.py:32] [ERROR]	got a fake error
[2020-01-07 22:57:44,929] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:00:34,264] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:01:20,896] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:04:08,309] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:11:27,809] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:13:20,165] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:15:50,982] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:16:28,267] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:18:28,733] [app.py:45] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:21:01,252] [app.py:44] [INFO]	init db sqlite:///E:\课程\Stanford\quarter5\researchlink\app.db
[2020-01-07 23:49:40,895] [app.py:44] [INFO]	init db sqlite:////home/wzq/Work/researchlink/app.db
[2020-01-08 00:02:22,238] [app.py:44] [INFO]	init db sqlite:////home/wzq/Work/researchlink/app.db
[2020-01-08 00:02:42,415] [app.py:44] [INFO]	init db sqlite:////home/wzq/Work/researchlink/app.db
[2020-01-08 00:02:52,727] [app.py:44] [INFO]	init db sqlite:////home/wzq/Work/researchlink/app.db
[2020-01-08 00:03:18,081] [app.py:44] [INFO]	init db sqlite:////home/wzq/Work/researchlink/app.db
